### DISCUSSION TITLE
The cost of using in June has been billed twice. Please help.

### DISCUSSION THREAD

**Discussion by tonyfromundefined** [vercel/community#3129] [Bug Report]
### Summary

I submitted a support ticket last Sunday (June 25th), but haven't received a response yet, so I'm posting it on the GitHub Discussion. (Case Number: 00143103)

- ‘17D98245-0034’ and ‘17D98245-0035’ invoices were paid twice with exactly the same content.
- It seems that the payment was made incorrectly due to an error, so please cancel one of the two as soon as possible

<img width="760" alt="Screenshot 2023-06-25 오전 10 40 11" src="https://github.com/vercel/community/assets/20325202/fd0688a6-3a2e-4713-b311-3a8d5b120f55">

### Steps to Reproduce

I do not know how to reproduce the billing issue.

**Comment by amyegan** [comment]
Thanks for reaching out about this, @tonyfromundefined. I found your support case and reached out to our billing team about it. They will review it today, so you can expect to get an update on the case soon 🙇
